requirements/requirements.txt

```diff
@@ -13,7 +13,7 @@ dj-database-url==0.4.1
 django-appconf==1.0.2     # via django-axes
 django-axes==4.1.0
 django-bootstrap-form==3.3
-django-cors-headers==1.1.0
```

Comment:
Are there other updates (e.g. `pip-compile --upgrade`) that we could easily roll into this?

Reply:
I believe that there will be more updates, and a good idea to roll them in now. Will try this and retest afterward.

Reply:
23 packages were updated to the latest version via the "pip-compile --upgrade" results. Two packages had issues with their latest versions, so used an older version for those. See top level notes for details.